src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$month-length: 30;
```

Comment:
this should be modified in browser instead of code - please implement modifier - readme point 4